tgui/packages/tgui/interfaces/TramController.tsx

```diff
@@ -0,0 +1,279 @@
+                  color={controllerActive ? 'blue' : 'good'}>
+                  {controllerActive ? 'Processing' : 'Ready'}
+                </LabeledList.Item>
+                <LabeledList.Item
+                  label="Mechanical Status"
+                  color={controllerOperational ? 'good' : 'bad'}>
+                  {controllerOperational ? 'Normal' : 'Fault'}
+                </LabeledList.Item>
+                <LabeledList.Item
+                  label="Processor Status"
+                  color={recoveryMode ? 'average' : 'good'}>
+                  {recoveryMode ? 'Overload' : 'Normal'}
+                </LabeledList.Item>
+                <LabeledList.Item label="Processor Load">
+                  <ProgressBar
+                    value={currentLoad}
+                    minValue={0}
+                    maxValue={15}
+                    ranges={{
+                      good: [-Infinity, 5],
+                      average: [5, 7.5],
+                      bad: [7.5, Infinity],
+                    }}
+                  />
+                </LabeledList.Item>
+                <LabeledList.Item label="Current Speed">
+                  <ProgressBar
+                    value={currentSpeed}
+                    minValue={0}
+                    maxValue={32}
+                    ranges={{
+                      good: [28, Infinity],
+                      average: [24, 28],
+                      bad: [0.1, 24],
+                      white: [-Infinity, 0],
+                    }}>
+                    {toFixed(currentSpeed * 2.25, 0) + ' km/h'}
+                  </ProgressBar>
+                </LabeledList.Item>
+              </LabeledList>
+            </Section>
+            <Section title="Location Data">
+              <LabeledList>
+                <LabeledList.Item label="Direction">
+                  {travelDirection === 4 ? 'Outbound' : 'Inbound'}
+                </LabeledList.Item>
+                <LabeledList.Item
+                  label="Idle Platform"
+                  color={controllerActive ? '' : 'blue'}>
+                  {idlePlatform}
+                </LabeledList.Item>
+                <LabeledList.Item
+                  label="Destination Platform"
+                  color={controllerActive ? 'blue' : ''}>
+                  {destinationPlatform}
+                </LabeledList.Item>
+              </LabeledList>
+            </Section>
+          </Flex.Item>
+          <Flex.Item width={480} px={0.5}>
+            <Section title="Controls">
+              <NoticeBox>
+                Nanotrasen is not responsible for any injuries or fatalities
+                caused by usage of the tram.
+              </NoticeBox>
+              <Button
+                icon="square"
+                color="bad"
+                my={1}
+                lineHeight={2}
+                minWidth={7}
+                minHeight={2}
+                textAlign="center"
+                content="E-Stop"
```

Comment:
Please use these as the child rather than using content
`<Button>E-Stop</Button>`

Reply:
Not quite sure why I did it like that anyways.

Reply:
suggestions, maybe! content was the old method. sometimes causes issues with spacing